.env

```diff
@@ -0,0 +1 @@
+EXTEND_ESLINT=true
```

Comment:
This is to make the "errors" for breaking various hooks checks into "warnings" - it is inconvenient because it either runs a hook for vitalik or the new tokens which eslint considers 'risky'.